static/app/views/detectors/components/forms/metric.tsx

```diff
@@ -0,0 +1,642 @@
+  flex-direction: column;
+  gap: ${space(4)};
+  padding: ${space(4)};
+`;
+
+const FirstRow = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  gap: ${space(1)};
+  border-bottom: 1px solid ${p => p.theme.border};
+`;
+
+const DetectColumn = styled(Flex)`
+  flex-grow: 1;
+`;
+
+const StyledSelectField = styled(SelectField)`
+  width: 180px;
+  padding: 0;
+  margin: 0;
+
+  > div {
+    padding-left: 0;
+  }
+`;
+
+const StyledMemberTeamSelectorField = styled(SentryMemberTeamSelectorField)`
+  padding-left: 0;
+`;
+
+const VisualizeField = styled(SelectField)`
+  flex: 2;
+  padding-left: 0;
+  padding-right: 0;
+  margin-left: 0;
+  border-bottom: none;
+`;
+
+const ChartContainer = styled('div')`
+  background: ${p => p.theme.background};
+  width: 100%;
+  border-bottom: 1px solid ${p => p.theme.border};
+  padding: 24px 32px 16px 32px;
+`;
+
+const AggregateField = styled(SelectField)`
+  width: 120px;
+  margin-top: auto;
+  padding-top: 0;
+  padding-left: 0;
+  padding-right: 0;
+  border-bottom: none;
+
+  > div {
+    padding-left: 0;
+  }
+`;
+
+const DirectionField = styled(SelectField)`
+  width: 16ch;
+  padding: 0;
+  margin: 0;
+  border-bottom: none;
+
+  > div {
+    padding-left: 0;
+  }
+`;
+
+const MonitorKindField = styled(SegmentedRadioField)`
+  padding-left: 0;
+  padding-block: ${space(1)};
+  border-bottom: none;
+  max-width: 840px;
+
+  > div {
+    padding: 0;
+  }
+`;
+const ThresholdField = styled(NumberField)`
+  padding: 0;
+  margin: 0;
+  border: none;
+
+  > div {
+    padding: 0;
+    width: 10ch;
+  }
+`;
+
+const ChangePercentField = styled(NumberField)`
+  padding: 0;
+  margin: 0;
+  border: none;
+
+  > div {
+    padding: 0;
+    max-width: 10ch;
+  }
+`;
+
+const MutedText = styled('p')`
+  color: ${p => p.theme.text};
+  padding-top: ${space(1)};
+  margin-bottom: ${space(1)};
+  border-top: 1px solid ${p => p.theme.border};
+`;
+
+function FilterField() {
+  return (
+    <Flex column gap={space(0.5)} style={{paddingTop: 16, flex: 1}}>
+      <span>Filter</span>
+      <SearchQueryBuilder
+        initialQuery=""
+        filterKeySections={FILTER_KEY_SECTIONS}
+        filterKeys={FILTER_KEYS}
+        getTagValues={getTagValues}
+        searchSource="storybook"
+      />
+    </Flex>
+  );
+}
+
+const getTagValues = (): Promise<string[]> => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(['foo', 'bar', 'baz']);
+    }, 500);
+  });
+};
+
+const FILTER_KEYS: TagCollection = {
+  [FieldKey.ASSIGNED]: {
+    key: FieldKey.ASSIGNED,
+    name: 'Assigned To',
+    kind: FieldKind.FIELD,
+    predefined: true,
+    values: [
+      {
+        title: 'Suggested',
+        type: 'header',
+        icon: null,
+        children: [{value: 'me'}, {value: 'unassigned'}],
+      },
+      {
+        title: 'All',
+        type: 'header',
+        icon: null,
+        children: [{value: 'person1@sentry.io'}, {value: 'person2@sentry.io'}],
```

Comment:
not familiar with how this works-- are these temp values that we need to make an API query for later?